### PR TITLE
Add FaultInjection as a separate test category

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

**Review by timtay-microsoft — APPROVED** [review]

File: build.ps1 [hunk 1]
```diff
@@ -62,7 +62,7 @@ Param(
     [switch] $clean,
     [switch] $build,
     [switch] $unittests,
-    [switch] $prtests,
```
Comment (drwill-ms) [inline build.ps1:65]: I think PR Tests was a better name for how we're using it. They aren't BVTs, strictly speaking. Our integration build and nightly builds run all the tests that validate the build.

**Review by drwill-ms — APPROVED** [review]

File: build.ps1 (hunk 1 above)
Comment (abhipsaMisra) [reply build.ps1:65]: I don't mind changing it back to PR tests.

**Pull request merged**